### PR TITLE
Main section

### PR DESCRIPTION
**Pull request by damfinkel**
## Summary

Main section layout. Also defined the containers for the two sections of the app. Their sizes might change in the future.

When the page's width is lower than 1000px, the phone and separator dissapear.

## Trello card

https://trello.com/c/9RVfCJ14/1-maquetar-la-seccion-pricipal-de-la-pagina

## Screenshots

screen shot 2017-10-20 at 3 49 15 pm
screen shot 2017-10-20 at 3 49 25 pm

### CODE REVIEW

File: src/training.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+
+<html>
```
Comment (SKOLZ) [inline src/training.html]: remove the empty lines, it's easier to find an incorrect tab and check where an element closes that way

File: src/training.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+    <title>The Cheese Guardian</title>
+    <link rel="stylesheet" href="training.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+  </head>
+
+  <body>
+
+    <main class="main">
```
Comment (SKOLZ) [inline src/training.html]: this container is not necesary, you can remove it and simplify the DOM

File: src/training.scss [hunk 3]
```diff
@@ -0,0 +1,56 @@
+$cream-can: #F5C85F;
```
Comment (SKOLZ) [inline src/training.scss]: create a `colors.scss` file and add this variables there, then import them in this file

File: src/training.scss [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+body {
+  display: flex;
+  flex-direction: column;
+}
+
+.main {
+  display: flex;
+  flex-direction: column;
+  flex-grow: 8;
```
Comment (SKOLZ) [inline src/training.scss]: why 8 haha, you only need 1. add that in the first section

File: src/training.scss [hunk 5]
```diff
@@ -0,0 +1,56 @@
+$cream-can: #F5C85F;
+$dusty-grey: #EEE;
+
+html, body {
```
Comment (SKOLZ) [inline src/training.scss]: to do this you should add a reset, you can read more on that [here](http://cssreset.com/what-is-a-css-reset/).

Use our reset, you can find it in the techguides in `/frontend/layout` with the name `the equalizer`

File: src/training.scss [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+  min-height: 100vh;
+  margin: 0;
+}
+
+body {
```
Comment (SKOLZ) [inline src/training.scss]: style classes instead of elements

File: src/training.scss [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+  }
+}
+
+.features-container {
+  background-color: $dusty-grey;
+  flex-grow: 2;
+}
+
+.logo-separator {
+  background: linear-gradient(white, $cream-can, white);
```
Comment (SKOLZ) [inline src/training.scss]: use a color variable for white

File: src/training.scss [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+    .logo-separator {
+      display: none;
+    }
+
+  }
+}
+
+.features-container {
+  background-color: $dusty-grey;
+  flex-grow: 2;
```
Comment (SKOLZ) [inline src/training.scss]: you don't need this, just let the first section grow

File: src/training.scss (hunk 5 above)
Comment (damfinkel) [reply src/training.scss]: I added the reset and removed this, but it broke stuff

File: src/training.scss (hunk 6 above)
Comment (damfinkel) [reply src/training.scss]: You told me to do this. I need some kind of container of the 3 main divs so I can use flex. Is there another way to do it?

File: src/training.scss (hunk 5 above)
Comment (damfinkel) [reply src/training.scss]: Should I add these lines to that file?

File: src/training.scss (hunk 6 above)
Comment (SKOLZ) [reply src/training.scss]: you should add a class to the body and style that class.

File: src/_colors.scss [hunk 9]
```diff
@@ -0,0 +1,3 @@
+$cream-can: #F5C85F;
+$dusty-grey: #EEE;
+$white: white;
```
Comment (SKOLZ) [inline src/_colors.scss]: use hex notation to be consistent.
```
$white: #FFF;
```

**Pull request merged**